### DISCUSSION TITLE
RubyKaigi 2024の"様子"がわかるような記事を残したい

### DISCUSSION THREAD

**Discussion by neko314** [rubima/magazine.rubyist.net#491] [要望・アイデア]
アイディアがまとまりきらないのですが、思いはあるのでDiscussion立ち上げてみます。

私個人として、数年前からるびまを読むようになり、初期のるびまの記事を読んでいて「この頃はこういう感じだったんだなあ」と伝わってくる点が大変趣深いと感じています。
先日RubyKaigi 2024が終了しましたが、資料などはスライドで残っていくかもしれませんが、それだけでは当日の雰囲気や熱気のようなものは記録に残せないので、るびまに一片でも記事として残せたらいいなと思っています。そして後日、数ヶ月後、数年後その記事を読んだときに、この時のことに思いを馳せられるようにしたいです。

ということを思っているのですが、どういう記事にすると良いかとか、誰に書いてもらうと良いだろうかとかの具体がノーアイディアです。

一人考えていたのはこの辺り
* 執筆してくださる人がいたらぜひお願いして、読み物な記事にする
* たくさんの方から写真を募ってフォトレポートのような記事にする
  * ちょっとしたエピソードも添えていただいたりすると味わい深くなりそう

**Comment by hsbt** [comment]
写真なら結構あります!

**Comment by neko314** [comment]
私自身の手元に写真が全然ないのですが、こういう感じのものをバーっと並べたいというのを簡単にdraft PRで作ってみました。
https://github.com/rubima/magazine.rubyist.net/pull/493#issuecomment-2135001111

ここから、@hsbt さんの良さげな写真を頂いて内容を充実させていく、XやSlackなどで、広く皆さんから募るのも良いかなーと構想中。

その場合注意するとしたら、白いストラップの写真撮影NGの方が写っていないこと・写っていたら顔を隠してあることのチェックくらいか。

**Comment by neko314** [comment]
https://github.com/rubima/magazine.rubyist.net/pull/493 を作った！
